src/extension/src/azure/azure-app-service/appServiceProvider.ts

```diff
@@ -118,8 +233,8 @@ export class AppServiceProvider {
     }
   }
 
-  private async generateValidASPName(name: string): Promise<string> {
-    let generatedName: string = NameGenerator.generateName(name, "asp");
+  public async generateValidASPName(name: string): Promise<string> {
```

Comment:
Why did you change the method from private to public? Is it being accessed out of this class?

Reply:
Yes, it is accessed out of this class now.

I initially thought that we need to create an App Service Plan for our user, but all that we need is a valid name. That's why we are using this method, and deleted this chunk: https://github.com/microsoft/WebTemplateStudio/pull/1014/files#diff-38b59b78a36d435c0373d85382d0d0e1L358-L368